test/lib/tasks/seeds_test.rb

```diff
@@ -70,13 +70,6 @@ class PulpProxyTest < SeedsTest
     end
   end
 
-  class PermissionsTest < SeedsTest
-    test "Make sure katello permissions got created exist" do
-      seed
-      assert Permission.pluck(:resource_type).grep(/Katello/).present?
-    end
-  end
```

Comment:
With this change, it looks like the permissions are no longer being seeded in the test env. Our controller tests create these permissions on the fly (which is why they are still passing without permissions in the db). We could do something in the future like [seeding them with a fixture file](https://github.com/theforeman/foreman/blob/develop/test/fixtures/permissions.yml).